Meme/Meme/NetworkClient.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  NetworkClient.swift
+//  Meme
+//
+//  Created by Mesfin Bekele Mekonnen on 7/29/19.
+//  Copyright © 2019 Mesfin Bekele Mekonnen. All rights reserved.
+//
+
+import Foundation
+
+class NetworkClient {
+
```

Comment:
this is probably where we should add 
```swift
func getMemeTemplates(...)
``` 
which can call getTemplates and then perform the conversion to meme or MemeTemplate etc

Reply:
@vhart Hmm, I haven't really defined a `MemeTemplate` data model right now. I'm down to just Meme, and Template. What's the difference between a MemeTemplate and a Template?

Reply:
Oh sorry @mesbekmek didn’t mean to complicate things...yet lol. I was imagining that the templates would be a MemeTemplate and the Meme would be the template + text top and bottom + meta data. Go with your gut we can refactor later if needed.

Reply:
Ah okay, that makes sense. How about we do that in an upcoming PR. I wanna merge this one first. I'll make the other changes you suggested in an upcoming commit to this PR, and if it's in a state you think is good enough we can merge this PR, and I can work on what you've described here.

Reply:
Sounds like a plan